src/avram/queryable.cr

```diff
@@ -99,7 +99,8 @@ module Avram::Queryable(T)
   end
 
   def order_by(column, direction) : self
-    query.order_by(column, direction)
+    direction = Avram::OrderBy::Direction.parse(direction.to_s)
```

Comment:
Could we change this to pass direction as-is to `OrderBy`? I think `order_by` isn't typically used directly so maybe that'll work?

Also how does `parse` work? Is that just something you get with Crystal enums?

Reply:
We can't because enums only take symbol literals. So passing it down actually throws an exception. `parse` is just on enum members. It takes a string and returns the enum or throws an exception. So this would catch if someone didn't pass the correct symbol

Reply:
ref: https://gitter.im/crystal-lang/crystal?at=5d76690d6e889c4bbdc77c63

Reply:
Gotcha! And that is cool about parse!

Reply:
Yeah, we can catch that. 

> Unknown enum Avram::OrderBy::Direction value: sideways (ArgumentError)

Reply:
Actually I think that is pretty good. I think we can keep that and just append a message: `Unknown enum Avram::OrderBy::Direction value: sideways` + `"Accepted values are: asc, desc" or something. What do you think of that?

Reply:
I'm cool with that! 👍